packages/openapi-to-graphql/src/index.ts

```diff
@@ -364,12 +383,70 @@ async function translateOpenAPIToGraphQL(
           mutationFields[saneFieldName] = field
         }
       }
+    } else {
+      // handle subscriptions from operation.callbacks
+      // 1) cbName would be the subscription field name
+      // each paths contained in operation.callbacks[cbName]
+      // would be a channel to subscribe on the resolver
+      // but if callback object contains several operations
+      // how to be sure that the returned Graphql type would be the same ?
+      // By "forcing' a common response schema for every operation within the CB ?
```

Comment:
We encountered a similar problem when creating `Query` and `Mutation` fields, which may have multiple successful response objects based on the HTTP code. In that case, we simply picked the first successful response object. Perhaps there are ways to improve this in the future. 

The other solutions would be try to create a union (as we do when resolving `oneOf`) or creating an object type with the superset of all fields (assuming that all response are objects as we do when resolving `anyOf`) or defaulting onto a JSON type. 

For now, I think we should just pick an arbitrary one and throw a warning.